docs/components/popovers.md

```diff
@@ -43,9 +43,9 @@ When you have some styles on a parent element that interfere with a popover, you
 
 {% highlight js %}
 $(function () {
-  $('.example-popover').popover(
+  $('.example-popover').popover({
     container: 'body'
```

Comment:
Ah, ok. That's an oversight on my part. Wasn't fully aware that there was a difference between "JSON" and "Javascript Object Literal". Sorry about that :+1: